test/test_popupform.js

```
@@ -140,32 +152,32 @@ const mockApp = {
+    makeAvatarUrl: params => {
       return params.avatarUrl || '';
-    }
-  }
+    },
+  },
 };
 
-// Execute the PopupViewForm code
-eval(popupViewFormCode);
+// Execute the PopupForm code
+eval(popupFormCode);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)